code/game/machinery/turnstile.dm

```diff
@@ -0,0 +1,89 @@
+/obj/machinery/turnstile
+	name = "turnstile"
+	desc = "A mechanical door that permits one-way access and prevents tailgating."
+	icon = 'icons/obj/turnstile.dmi'
+	icon_state = "turnstile_map"
+	power_channel = ENVIRON
+	density = FALSE
+	armor = list(melee = 50, bullet = 50, laser = 50, energy = 50, bomb = 10, bio = 100, rad = 100, fire = 90, acid = 70)
+	anchored = TRUE
+	use_power = FALSE
+	idle_power_usage = 2
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	layer = OPEN_DOOR_LAYER
+	var/cuts_off_zipties = FALSE

```

Comment:
I'd remove this var altogether. If you can drag someone through the stuff you can take off the cuffs manually and there's nothing to indicate whether the turnstile will do that for you.